### DISCUSSION TITLE
showing github.token(probably same as GITHUB_TOKEN) is intentional?

### DISCUSSION THREAD

**Discussion by msroz** [community/community#25897] [Actions]
<p>Hi</p>
<p>I’ve just started trying GitHub Actions.</p>
<p>I’m wondering if it’s intentional that github.token(probably same as GITHUB_TOKEN) in <a href="https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions#contexts" rel="nofollow noopener">this page</a> is shown in log view. is it unsafe? </p>
<p>thanks:)</p>
<p><img src="https://user-images.githubusercontent.com/1856081/181093858-e967c807-bec8-40d5-b1bd-9a4d70103390.png" alt="スクリーンショット 2019-09-19 16.57.40.png" data-base62-sha1="4lku3ymG1BeVw9x1bCfsqliAs3" width="511" height="105"> <img src="https://user-images.githubusercontent.com/1856081/181093859-98746b56-94da-4de6-a14e-5300de8c340a.png" alt="スクリーンショット 2019-09-19 17.00.00.png" data-base62-sha1="qUBo2VRn3t6Xics9gDg75bOx1Af" width="588" height="185"></p>

**Comment by ethomson** [comment, marked as answer]
<p>Interestingly, this was not the same as secrets.GITHUB_TOKEN - your workflow is passing environment variables <code>token1</code> and <code>token2</code> but you’re evaluating <code>TOKEN1</code> and <code>TOKEN2</code>.  Environment variables are case sensitive (on POSIX platforms) so they were both empty and that’s why they were the same in your test.</p>
<p>But that’s an aside.  This was part of a change that we made this morning to how temporary secrets are managed and renewed.  Prior to this, <code>github.token</code> and <code>secrets.github_token</code> were the same value.  With this change, we were separating them into distinct values.  In this case, we should have been masking the value of the <code>github.token</code> like we were for the <code>secrets.github_token</code>.</p>
<p>We’ve rolled back this change, so now the value of those two variables are the same again, which means that you should not be able to see <code>github.token</code> in any log output, it should be masked.  (But also note that any tokens that were visible in log output were time limited and have already expired.)</p>
<p>Thanks for letting us know.</p>

**Comment by msroz** [comment]
<p>Thanks for replying!</p>
<p>I was mistaken that environment variables are case insensitive as I read <a href="https://help.github.com/ja/articles/virtual-environments-for-github-actions#environment-variables" rel="nofollow noopener">Japanese translation (maybe a bit older than English one).</a></p>
<p>I also understood the details about <code>github.token</code>  and  <code>secrets.GITHUB_TOKEN</code>.</p>
<p>Thank you!!</p>
